### DISCUSSION TITLE
Getting a kitty dock panel similar to one shown in the kittens/panel page

### DISCUSSION THREAD

**Discussion by neelchakrabrty** [kovidgoyal/kitty#6924] [Q&A]
Hello, can someone please tell me how to get a panel [similar to one  shown in the screenshot of the kitty kittens/panel page?](https://sw.kovidgoyal.net/kitty/kittens/panel/) I am quite new to this and can't figure out how to get the colored angled blocks as shown in the screenshot,or how to get the spacing between left and right modules.

**Comment by kovidgoyal** [comment]
On Sun, Dec 17, 2023 at 04:50:58AM -0800, stallonen wrote:
> Hello, can someone please tell me how to get a panel [similar to one  shown in the screenshot of the kitty kittens/panel page?](https://sw.kovidgoyal.net/kitty/kittens/panel/) I am quite new to this and can't figure out how to get the colored angled blocks as shown in the screenshot,or how to get the spacing between left and right modules. 

See for example how powerline works, basically you output the angle
chars with appropriate foreground and background colors.

**Comment by neelchakrabrty** [comment]
I looked into powerline and it seems it's what I need. Quick question, how do I actually launch powerline? I went through the documentation, but there's no guide on how to launch powerline. I saw how to integrate it with window managers, and I assume that's not what is happening in the screenshot I mentioned.
